doc/deployment-schema.json

```diff
@@ -0,0 +1,310 @@
+                                                },
+                                                "settings": {
+                                                    "$ref": "#/definitions/moduleSettings"
+                                                }
+                                            }
+                                        }
+                                    },
+                                    "additionalProperties": false
+                                }
+                            }
+                        }
+                    }
+                },
+                "$edgeHub": {
+                    "type": "object",
+                    "title": "Configuration for the edgeHub module",
+                    "required": [
+                        "properties.desired"
+                    ],
+                    "properties": {
+                        "properties.desired": {
+                            "type": "object",
+                            "required": [
+                                "schemaVersion",
+                                "routes"
+                            ],
+                            "properties": {
+                                "schemaVersion": {
+                                    "type": "string",
+                                    "examples": [
+                                        "1.0"
+                                    ]
+                                },
+                                "routes": {
+                                    "type": "object",
+                                    "patternProperties": {
+                                        "^.+$": {
+                                            "type": "string",
+                                            "examples": [
+                                                "FROM /* INTO $upstream"
+                                            ]
+                                        }
+                                    }
+                                },
+                                "storeAndForwardConfiguration": {
+                                    "type": "object",
+                                    "required": [
+                                        "timeToLiveSecs"
+                                    ],
+                                    "properties": {
+                                        "timeToLiveSecs": {
+                                            "type": "integer",
+                                            "examples": [
+                                                7200
+                                            ]
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "patternProperties": {
+                "^.+$": {
```

Comment:
module twin key should be same pattern as module names in line 147 ? "^[a-zA-Z0-9_.-]*$"

Reply:
yes, updated